### DISCUSSION TITLE
Can gsconnect support device discovery via Bluetooth?

### DISCUSSION THREAD

**Discussion by fxzxmicah** [andyholmes/gnome-shell-extension-gsconnect#1792] [Q&A]
Since I am on a public network, the IP on the device will change so I can't connect directly. But the Bluetooth connection between the two will not change. So is it possible to communicate via Bluetooth or just exchange IP with each other via Bluetooth?

**Comment by ferdnyc** [comment]
As noted in another discussion, [KDE Connect now supports Bluetooth](https://linuxiac.com/kde-plasma-6-desktop-released/#h-kde-connect), not just for discovery but for all communication between two devices. (Albeit over a much slower and less reliable connection.)

And potentially even more useful in your situation, it also now supports mDNS, which would theoretically allow you to pair your device by local hostname so it'll automatically be followed even when the IP changes.

GSConnect, unfortunately, currently supports neither of those features, and so far no developers have expressed any intention to work on developing those features.

**Reply by fxzxmicah** [reply]
Thanks.

**Comment by GerardPolloRebozado** [comment]
have there been any progress on this? or still no intention on developing this?
